tests/test_models.py

```diff
@@ -1,31 +1,103 @@
-    npt.assert_array_equal(daily_mean(test_input), test_result)
+    npt.assert_array_equal(daily_mean(test), expected)
 
 
-def test_daily_mean_integers():
-    """Test that mean function works for an array of positive integers."""
-    from inflammation.models import daily_mean
+@ pytest.mark.parametrize(
+    "test, expected",
+    [
+        ([[0, 0], [0, 0], [0, 0]], [0, 0]),
+        ([[1, 2], [3, 4], [5, 6]], [5, 6]),
+    ]
+)
+def test_daily_max_integers(test, expected):
+    "test the max function works for an array of positive integers"
+    from inflammation.models import daily_max
+
+    # Need to use Numpy testing functions to compare arrays
+    npt.assert_array_equal(daily_max(test), expected)
+
 
-    test_input = np.array([[1, 2],
-                           [3, 4],
-                           [5, 6]])
-    test_result = np.array([3, 4])
+@ pytest.mark.parametrize(
+    "test, expected",
+    [
+        ([[0, 0], [0, 0], [0, 0]], [0, 0]),
+        ([[1, 2], [3, 4], [5, 6]], [1, 2]),
+    ]
+)
+def test_daily_min_integers(test, expected):
+    "test the max function works for an array of positive integers"
+    from inflammation.models import daily_min
 
     # Need to use Numpy testing functions to compare arrays
-    npt.assert_array_equal(daily_mean(test_input), test_result)
+    npt.assert_array_equal(daily_min(test), expected)
+
+
+def test_daily_min_string():
+    from inflammation.models import daily_min
+
+    with pytest.raises(TypeError):
+        error_expected = daily_min([['abd', 'ads'], ['asd', 'auhs']])
+
+
+@pytest.mark.parametrize(
+    "test, expected, expect_raises",
+    [
+        ([[0, 0, 0], [0, 0, 0], [0, 0, 0]],
+         [[0, 0, 0], [0, 0, 0], [0, 0, 0]],
+         None),
+
+        ([[1, 1, 1], [1, 1, 1], [1, 1, 1]],
+         [[1, 1, 1], [1, 1, 1], [1, 1, 1]],
+         None),
+
+        ([[1, 2, 3], [4, 5, 6], [7, 8, 9]],
+         [[0.33, 0.67, 1], [0.67, 0.83, 1], [0.78, 0.89, 1]],
+         None),
+
+        ([[-1, 2, 3], [4, 5, 6], [7, 8, 9]],
+         [[0.33, 0.67, 1], [0.67, 0.83, 1],
+         [0.78, 0.89, 1]],
+         ValueError),
+    ]
+)
+def test_patient_normalise(test, expected, expect_raises):
+    '''Test normalisations works'''
+    from inflammation.models import patient_normalise
+
+    if expect_raises is not None:
+        with pytest.raises(expect_raises):
+            npt.assert_almost_equal(patient_normalise(
+                np.array(test)), np.array(expected), decimal=2)
+    else:
+        npt.assert_almost_equal(patient_normalise(
+            np.array(test)), np.array(expected), decimal=2)
+
+@pytest.mark.parametrize(
+    "test, expected",
+    [
+        ([[0, 0], [0, 0], [0, 0]], [0, 0]),
+        ([[1, 2], [3, 4], [5, 6]], [1.633, 1.633]),
+    ]
+)
+def test_standard_devation(test, expected):
+    """test for standard deviation"""
+    from inflammation.models import standard_deviation
 
+    npt.assert_almost_equal(standard_deviation(test), expected, decimal=7)
```

Comment:
test expected is to 3 decimal places this is to 7
```suggestion
    npt.assert_almost_equal(standard_deviation(test), expected, decimal=3)
```